videos/README.md

```diff
@@ -0,0 +1,48 @@
+# Video Workflow
+
+This document describes the components of the video workflow for OCW.
+
+**SECTIONS**
+
+1. [Overview](#overview)
+1. [Google Drive Sync and AWS Transcoding](#google-drive-sync-and-aws-transcoding)
+1. [YouTube Submission](#youtube-submission)
+1. [Captioning and 3Play Transcript Request](#captioning-and-3play-transcript-request)
+1. [Completing the Workflow](#completing-the-workflow)
+1. [Management Commands](#management-commands)
+
+# Overview
+
+This assumes that AWS Media Convert, Google Drive sync, YouTube integration, and 3Play submission are all enabled, which is required for the video workflow.
```

Comment:
Not really necessary, but might be nice to have links to the relevant sections in the other README here.